src/python/grpcio_tests/tests/interop/client.py

```diff
@@ -85,30 +89,33 @@ def _create_call_credentials(args):
 def get_secure_channel_parameters(args):
     call_credentials = _create_call_credentials(args)
 
-    if args.use_test_ca:
-        root_certificates = resources.test_root_certificates()
-    else:
-        root_certificates = None  # will load default roots.
-
-    channel_credentials = grpc.ssl_channel_credentials(root_certificates)
-    if call_credentials is not None:
-        channel_credentials = grpc.composite_channel_credentials(
-            channel_credentials, call_credentials)
-
     channel_opts = None
-    if args.server_host_override:
-        channel_opts = ((
-            'grpc.ssl_target_name_override',
-            args.server_host_override,
-        ),)
+    if args.use_tls:
+        if args.use_test_ca:
+            root_certificates = resources.test_root_certificates()
+        else:
+            root_certificates = None  # will load default roots.
+
+        channel_credentials = grpc.ssl_channel_credentials(root_certificates)
+        if call_credentials is not None:
+            channel_credentials = grpc.composite_channel_credentials(
+                channel_credentials, call_credentials)
+
+        if args.server_host_override:
+            channel_opts = ((
+                'grpc.ssl_target_name_override',
+                args.server_host_override,
+            ),)
+    else args.use_alts:
+        channel_credentials = grpc.alts_channel_credentials(['svc_account1@gmail.com'])
```

Comment:
should this service account be hardcoded, rather than using the VM's default account?